woocommerce-stats-by-currency.php

```diff
@@ -133,3 +133,67 @@ function wsbc_add_stats() {
 
 }
 add_action( 'woocommerce_after_dashboard_status_widget', 'wsbc_add_stats' );
+
+/**
+ * Check required plugins
+ *
+ * @return void
+ */
+
+function wsbc_required_plugins() {
+	$activated = (array) get_option( 'active_plugins', array() );
```

Comment:
```suggestion
	$activated = apply_filters( 'active_plugins', (array) get_option( 'active_plugins', array() ) );
```

Let's respect the data filtering.